src/pages/tasks/NewTaskDetailsPage.tsx

```diff
@@ -0,0 +1,128 @@
+import ExpensiMark from 'expensify-common/lib/ExpensiMark';
+import React, {useEffect, useState} from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx';
+import FormProvider from '@components/Form/FormProvider';
+import InputWrapper from '@components/Form/InputWrapper';
+import type {FormOnyxValues} from '@components/Form/types';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import TextInput from '@components/TextInput';
+import useAutoFocusInput from '@hooks/useAutoFocusInput';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import * as TaskActions from '@userActions/Task';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import INPUT_IDS from '@src/types/form/NewTaskForm';
+import type {Task} from '@src/types/onyx';
+
+type NewTaskDetailsPageProps = {
+    /** Grab the Share title of the Task */
+    task: OnyxEntry<Task>;
+};
+
+const parser = new ExpensiMark();
+
+function NewTaskDetailsPage({task}: NewTaskDetailsPageProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const [taskTitle, setTaskTitle] = useState<string>(task?.title ?? '');
+    const [taskDescription, setTaskDescription] = useState<string>(task?.description ?? '');
```

Comment:
Same. Should be inferred.
```suggestion
    const [taskTitle, setTaskTitle] = useState(task?.title ?? '');
    const [taskDescription, setTaskDescription] = useState(task?.description ?? '');
```